Lib/test/test_cmath.py

```diff
@@ -2,7 +2,8 @@
 from test.test_math import parse_testfile, test_file
 import test.test_math as test_math
 import unittest
-import cmath, math
+import cmath
```

Comment:
did you run a formatter on the file? We follow the same exact structure as CPython's test files (except for cases where an error justifies the change). If you can, revert to the previous formatting.

Reply:
ok i will revert back to old formatting